src/pages/instances/InstanceDetail.tsx

```diff
@@ -71,18 +71,31 @@ const InstanceDetail: FC = () => {
     <main className="l-main">
       <div className="p-panel instance-detail-page">
         <div className="p-panel__header">
+          <h1 className="u-off-screen">{name}</h1>
           {instance ? (
-            <List
-              className="p-panel__title"
-              inline
-              items={[
-                <span key="name">{name}</span>,
-                <i key="status" className="p-text--small">
-                  {instance.status}
-                </i>,
-                <InstanceStateActions key="state" instance={instance} />,
-              ]}
-            />
+            <div className="p-panel__title">
+              <nav
+                key="breadcrumbs"
+                className="p-breadcrumbs"
+                aria-label="Breadcrumbs"
+              >
+                <ol className="p-breadcrumbs__items">
+                  <li className="p-breadcrumbs__item">
+                    <Link to={`/ui/${project}/instances`}>Instances</Link>
+                  </li>
+                  <li
+                    className="p-breadcrumbs__item instance-name u-truncate"
+                    title={name}
```

Comment:
The name can be up to 63 characters long. Considering M as the widest character, [this](https://lxd-ui-260.demos.haus/ui/default/instances/detail/MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM) is the worst it can get. We should find a way for long instance names not to break the layout. I think removing the `max-width` of the `p-panel__title` should fix in most cases, except there is a small range ~1100px width where it is still funny.

Reply:
Ok, I set the `max-width` of the `p-panel__title` here to `none`.

Reply:
Let's lower the max-width for the instance-name from `35vw` to `33vw`. Otherwise, it breaks into two lines at ~1100px screen width.

Reply:
![Screenshot 2023-03-31 at 9 24 45](https://user-images.githubusercontent.com/114913788/229040150-e24c896d-b038-4a4f-93d1-add64882d8c1.png)
e name at some point and maybe display the full one on a hover. Mobile users will only see it

Reply:
Done - thanks for the suggestion. Setting it to `33vw` fixes the problem indeed.